filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -442,7 +442,11 @@ DECL_DRIVER_API_N(readPixels,
         uint32_t, y,
         uint32_t, width,
         uint32_t, height,
-        backend::PixelBufferDescriptor&&, data)
+        backend::PixelBufferDescriptor&&, data,
+        backend::TextureSwizzle, r,
+        backend::TextureSwizzle, g,
+        backend::TextureSwizzle, b,
+        backend::TextureSwizzle, a)
```

Comment:
I think this is fine, I guess we can't re-use `ReadPixelsOptions` here because of DriverAPI macro magic, right?

Reply:
We could in theory, but we'd have to include the public `Renderer.h` header in `DriverAPI.inc`, which requires libbackend to depend on libfilament.

Reply:
The way to do this is to move the struct to DriverEnums.h, and do an alias in Renderer.h.
I've done this a lot before.